.travis.yml

```diff
@@ -15,7 +15,9 @@ env:
   - TEST_COMMAND=test EXTRA_FLAGS='' FEATURES='serde'
   # Tests building without std. We have to select a backend, so we select the one
   # most likely to be useful in an embedded environment.
-  - TEST_COMMAND=build EXTRA_FLAGS=--no-default-features FEATURES='u32_backend'
+  - TEST_COMMAND=build EXTRA_FLAGS='--no-default-features' FEATURES='u32_backend'
+  # Tests no_std+alloc usage using the most embedded-friendly backend
+  - TEST_COMMAND=test EXTRA_FLAGS='--lib --no-default-features' FEATURES='alloc u32_backend'
```

Comment:
The test suite passes with the 'alloc' feature, however for some reason `rand` isn't available in the test environment unless `std` is enabled?

Reply:
Hmm, I'm not sure what's happening there. Do we want to run tests without `std`? (Can tests be run without `std`?)

Reply:
`std` is still linked, but this is testing the configuration when the `std` feature is not enabled. It can and does work and pass (aside from the doc tests, hence the `--lib` flag).